et.json

```diff
@@ -0,0 +1,296 @@
+            "has-reminders" : "Sul on meeldetuletus selle asja kohta.",
+            "no-reminders" : "Meeldetuletused pole saadaval selle eseme kohta.",
+            "images" : "Pildid",
+            "images-icon" : "Ikoon",
+            "images-png" : "PNG",
+            "images-gallery" : "Galerii",
+            "images-featured" : "Esile toodud",
+            "interest-title" : "Huvi üle aja",
+            "interest-line1" : "See näitab huvi selle asja kohta üle aja, 100% tähendab, et ese on kõige populaarsem üle aja.",
+            "interest-line2" : "Andmed tänase kohta võivad olla mittetäielikud",
+            "occurrences-table" : {
+                "title" : "Poe Esinemisi",
+                "th-date" : "Kuupäev",
+                "th-days" : "Päeva Tagasi"
+            },
+            "set" : "Komplekt",
+            "set-name" : "Osa [name] komplektist",
+            "popular-title" : "Tänased populaarsed esemed"
+        },
+        "account" : {
+            "home" : {
+                "title" : "Andmed Sinu Kasutaja Kohta",
+                "error" : "Midagi läks valesti sinu kasutaja andmete laadimisel, palun proovi hiljem uuesti.",
+                "info" : {
+                    "title" : "Informatsioon",
+                    "line1" : "Sa oled sisse logitud [provider] kaudu [username]'na",
+                    "line2-api-yes" : "Sul on [juurdepääs api'le](/api/docs)",
+                    "line2-api-no" : "Sul ei ole juurdepääs api'le."
+                },
+                "migration" : {
+                    "title" : "Legaat Kasutajad",
+                    "waiting-line1" : "Me kolisime Discord'i, et sisse logida.",
+                    "waiting-line2" : "Kui sul on kasutaja aastast 2018, pead sa tooga sisse logima, et liituda sellega",
+                    "twitter-line1" : "Me kasutame nüüd Discord'i, et sisse logida. Sa kasutad praegu legaat kontot, mis võib tulevikus lõpetatud olla ning millel on limiteeritud funktsionaalsus.",
+                    "twitter-line2" : "Et liituda Discord'i kontoga, vajutage all olevale nupule"
+                },
+                "ratings" : {
+                    "title" : "Hinnangud",
+                    "none" : "Sul pole hinnanguid."
+                },
+                "destroy" : {
+                    "title" : "Hävita Konto",
+                    "line1" : "Kui sa enam ei soovi enda kontot, võid konto ja selle andmed eemaldada [siin](/account/destroy)"
+                }
+            },
+            "oauth" : {
+                "title" : "Logi sisse fnbr.co'sse",
+                "discord-button" : "Logi sisse Discord'iga",
+                "legacy-text" : "Teisel juhul, kui sul on legaat konto aastast 2018, mis vajab liitumist:",
+                "twitter-button" : "logi sisse Twitter'iga",
+                "privacy-text" : "Logides sisse, lubad meil kasutada teie kasutajaga seotud andmeid, nagu on mainitud meie puutumatusnormide kirjelduses](/privacy)."
```

Comment:
The left square bracket (`[`) is missing which breaks the link to the privacy policy. You may want to check the English language file for reference.